pyvelox/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+if(VELOX_BUILD_PYTHON_PACKAGE)
+  message("Creating pyvelox module")
+  include_directories(SYSTEM ${CMAKE_SOURCE_DIR})
+  add_definitions(-DCREATE_PYVELOX_MODULE)
+  # Define our Python module:
+  pybind11_add_module(
+    pyvelox MODULE
+    NO_EXTRAS # TODO: LTO crashes GCC9.2. File issues to pybind11
```

Comment:
This is added since linker time optimization seems to crash GCC before. So added this (https://pybind11.readthedocs.io/en/stable/cmake/index.html#pybind11-add-module).

may want to check again if this is still an issue (especially i suspect it should be fine only with type binding). Otherwise we can remove this to enable LTO